### PR TITLE
Fixed problem with calling unimported method in R.

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: configure.win [hunk 1]
```diff
@@ -47,9 +47,8 @@ fi
 ## Find the location of the MSBuild.exe executable.
 GET_MSBUILD_CMD=`echo ${RCLR_SRC_DIR}/get_msbuildpath.cmd | sed -e 's/\//\\\\/g'`
 # Do note that this one returns a slashified path already. See get_msbuildpath
-MSBUILD_EXE_SLASH=`cmd /c "$GET_MSBUILD_CMD"`
```
Comment (PavelBal) [inline configure.win:50]: No idea what's going on here, it will not compile with `cmd` now.

File: configure.win [hunk 2]
```diff
@@ -75,7 +74,7 @@ else
 	# create the RDll.lib and Mono.lib files for compiling with VC++. Do not assume they are set up (even if very likely for the CRAN winbuilder)
 	# Thanks to http://cygwin.com/ml/cygwin/2011-09/msg00232.html for how to call cmd files from sh.
 	CREATE_LIB_CMD=`echo ${RCLR_SRC_DIR}/create_lib.cmd | sed -e 's/\//\\\\/g'`
-	cmd /c "$CREATE_LIB_CMD"
+	"$CREATE_LIB_CMD"
```
Comment (PavelBal) [inline configure.win:77]: Same as above.

File: src/ClrFacade/RDotNetDataConverter.cs [hunk 3]
```diff
@@ -586,7 +586,7 @@ public Function CreateClrS4Object
             get
             {
                 if (createClrS4Object == null)
-                    createClrS4Object = engine.Evaluate("invisible(getCurrentConvertedObject)").AsFunction();
+                    createClrS4Object = engine.Evaluate("invisible(rClr::getCurrentConvertedObject)").AsFunction();
```
Comment (msevestre) [inline src/ClrFacade/RDotNetDataConverter.cs:589]: IS THAT IT?!?!? WOW

Comment (PavelBal) [inline src/ClrFacade/RDotNetDataConverter.cs:589]: Here is the sneaky call to the unimported method!

**Comment by msevestre** [comment]
Looks good to me. 
@PavelBal We need to update rClr release package right?

**Comment by PavelBal** [comment]
@Yuri05 I have the source (*.tar.gz) and the compiled (*.zip) windows packages for this. Don't have a Linux environment, though.

**Comment by PavelBal** [comment]
> We need to update rClr release package right?

@msevestre Yepp, I hope @Yuri05 will create a release for V10 (0.9 is also required for working with R4).

**Pull request merged**